### ISSUE TITLE
Is this project dead?

### ISSUE THREAD

**Issue by haught** [aacotroneo/laravel-saml2#260]
This project looks dead, aacotroneo appears to not touch this repo. Is there a fork that has an active development community?

**Comment by rroblik** [comment]
For sure it is dead :(

It *should* work as it under Laravel 9x but it should be officially forked/transferred @aacotroneo  

Thanks

**Comment by aacotroneo** [comment]
yeah, even though the library spirit was to stay small and only be a connector between OneLogin and Laravel, the big combination of versions and usecases makes it very hard to accept PRs without being deeply involved. the project lasted this long because of staying simple, but I understand some changes need to be made from time to time.

I'd be glad to advertise a new fork or to transfer to a new maintainer, please let me know how can I help.

**Comment by brezzhnev** [comment]
There is a fork I have created from this repository, we will maintain it as a part of our enterprise product: https://github.com/24Slides/laravel-saml2

**Comment by aacotroneo** [comment]
that's great @brezzhnev I linked your fork in the READ.me as an alternative. hopefully, it will work out well for everybody!
